examples/auto-test/start.sh

```diff
@@ -0,0 +1,28 @@
+echo "Run with parameter ci/release/normal!"
+if [ ! -d "pai/" ]; then
+    git clone -b yuqian/examples_fix https://github.com/Microsoft/pai.git
+    mv pai/examples/ . && rm -rf pai/
```

Comment:
If the pai dirrectory will be deleted. I suggest to clone the code to tmp dir.

Reply:
done